gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py

```diff
@@ -0,0 +1,315 @@
+import mock
+import unittest
+import subprocess
+
+from pecan import rest
+from pecan import set_config
+from pecan.testing import load_test_app
+
+mock_request = mock.Mock()
```

Comment:
Why is the mock_request instance needed? This is not used anywhere in the code. Can this be removed?